analysis/preregistration_template.Rmd

```diff
@@ -65,6 +66,7 @@ A static document for this type of process doesn't really make sense then. I pro
 ### 1.3.2 - Identify outcomes to evaluate decisions under scenarios
 
 *The outcomes should speak directly to the analytical objectives identified in 1.3.1. Outcomes could be qualitative or quantitative, and include descriptive statistic measures, satisficing measures or regret measures (Moallemi et al. 2019). For example, a model outcome in the top 10th percentile of worst conditions, or some minimum performance threshold for meeting a performance objective, or choosing a decision with the minimum regret or least amount of uncertainty.*
+To be honest, I don't know how I might apply this to my work or any future works. Maybe I do not have enough knowledge of this type of research to apply it to my basic ecology research. But this isn't to say it's not important, I just wonder if there's a way to make this accessible to non-desicion or modelling peeps. Like, how would I apply it to the woodland condition analysis? 
```

Comment:
I think perhaps this should be made optional. I guess it's only relevant if there's a specific decision under consideration, and you're using the model or analysis to simulate different future scenarios and evaluate different management interventions.